src/main/java/org/jline/utils/AttributedStyle.java

```diff
@@ -42,14 +36,22 @@
     static final int F_CROSSED_OUT  = 0x00000080;
     static final int F_FOREGROUND   = 0x00000100;
     static final int F_BACKGROUND   = 0x00000200;
+    static final int F_GENERIC_ESCAPE  = 0x00400;
 
-    static final int MASK           = 0x000003FF;
+    static final int MASK           = 0x000007FF;
 
     static final int FG_COLOR_EXP    = 16;
     static final int BG_COLOR_EXP    = 24;
     static final int FG_COLOR        = 0xFF << FG_COLOR_EXP;
     static final int BG_COLOR        = 0xFF << BG_COLOR_EXP;
 
+    public static final AttributedStyle DEFAULT = new AttributedStyle();
+    public static final AttributedStyle BOLD = DEFAULT.bold();
+    public static final AttributedStyle BOLD_OFF = DEFAULT.boldOff();
+    public static final AttributedStyle INVERSE = DEFAULT.inverse();
+    public static final AttributedStyle INVERSE_OFF = DEFAULT.inverseOff();
+    public static final AttributedStyle GENERIC_ESCAPE =
+        new AttributedStyle(F_GENERIC_ESCAPE, F_GENERIC_ESCAPE);
 
```

Comment:
This line is wrong.  It should be similar to the other ones, with an additional GENERIC_ESCAPE_OFF field, so that styles can be used with the builder style api and be chained.